.github/workflows/build.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
+
+# Usage:
+# ./.github/workflows/build.sh 
+
+# this script will modified upper directory istio-proxy repo
+# if self-used should attention that whether the upper directory have its own developing istio.proxy
+#first build should give permission to docker volumes
+#sudo chmod -R 777 /var/lib/docker/volumes
+
+UPDATE_BRANCH=${UPDATE_BRANCH:-"master"}
+cd ..
+rm -rf istio-proxy
```

Comment:
It will not removed automatically after build finished, to maintain the istio-proxy repo is latest, I think it is needed. It we moved from self-hosted runner to github-hosted runner it will be not needed.